src/pages/AppBody.tsx

```diff
@@ -19,7 +19,6 @@ export const BodyWrapper = styled.main<BodyWrapperProps>`
   margin-left: auto;
   margin-right: auto;
   z-index: ${Z_INDEX.default};
-  font-feature-settings: 'ss01' on, 'ss02' on, 'cv01' on, 'cv03' on;
```

Comment:
Why are these being turned off?

Reply:
this makes sense to turn off. they are tweaks for the font:
- ss = stylistic set
- cv = character variant

if we want the font to look as expected - the default variants - we should omit this line.

Reply:
they were causing the wrong looking `g`